### PR TITLE
gajim: update to 1.9.5.

### PR DESCRIPTION
**Pull request by lissine0**
* **python3-nbxmpp: update to 5.0.4.**
 
 * **gajim: update to 1.9.5.**
 

 #### Testing the changes
 
* I tested the changes in this PR: **YES**
 
 
#### Local build testing
 
* I built this PR locally for my native architecture, (x86_64-glibc)

Note: this PR depends on https://github.com/void-linux/void-packages/pull/52316 (and possibly https://github.com/void-linux/void-packages/pull/52315 as well)

### CODE REVIEW

**Comment by lissine0** [comment]
@Emru1 you may want to finish this PR
It builds and works correctly. But when enabling tests (`./xbps-src -Q pkg gajim`), it fails.
Do note that the way Gajim is built recently changed. See https://dev.gajim.org/gajim/gajim/-/commits/master/README.md

**Comment by lissine0** [comment]
Here's the test error:
```
=> gajim-1.9.4_1: installing host dependencies: gettext-0.22.5_1 python3-wheel-0.44.0_1 python3-build-1.2.2_1 python3-installer-0.7.0_2 farstream-0.2.9_4 gsound-1.0.3_1 gst-libav-1.24.6_1 gst-plugins-base1-1.24.6_1 gst-plugins-ugly1-1.24.6_1 gtksourceview4-4.8.4_1 libnm-1.48.10_1 libsecret-0.21.4_1 libsoup3-3.4.2_1 python3-Pillow-10.4.0_1 python3-SQLAlchemy2-2.0.23_1 python3-cairo-1.26.0_1 python3-css-parser-1.0.8_2 python3-dbus-1.3.2_2 python3-distro-1.9.0_1 python3-emoji-2.7.0_2 python3-gobject-3.48.2_1 python3-jeepney-0.8.0_3 python3-keyring-25.2.1_1 python3-nbxmpp-5.0.4_1 python3-netifaces-0.11.0_5 python3-omemo-dr-1.0.0_2 python3-openssl-24.2.1_1 python3-packaging-24.1_1 python3-precis-i18n-1.1.0_1 python3-pyasn1-0.5.1_1 python3-qrcode-7.4.2_1 python3-setuptools-75.1.0_1 python3-setuptools-75.1.0_1 python3-packaging-24.1_1 ...
=> gajim-1.9.4_1: running do-fetch hook: 00-distfiles ...
=> gajim-1.9.4_1: running do-extract hook: 00-distfiles ...
=> gajim-1.9.4_1: extracting distfile(s), please wait...
=> gajim-1.9.4_1: running do-patch hook: 00-patches ...
=> gajim-1.9.4_1: running pre-configure hook: 00-gnu-configure-asneeded ...
=> gajim-1.9.4_1: running pre-configure hook: 01-override-config ...
=> gajim-1.9.4_1: running pre-configure hook: 02-script-wrapper ...
=> gajim-1.9.4_1: running pre-build hook: 02-script-wrapper ...
=> gajim-1.9.4_1: running do_build ...
* Getting build dependencies for wheel...

Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/pyproject_hooks/_impl.py", line 402, in _call_hook
    raise BackendUnavailable(
pyproject_hooks._impl.BackendUnavailable: Cannot import 'setuptools.build_meta'

ERROR Backend 'setuptools.build_meta' is not available.
=> ERROR: gajim-1.9.4_1: do_build: 'python3 -m build --no-isolation --wheel ${make_build_args} ${make_build_target}' exited with 1
=> ERROR:   in do_build() at common/build-style/python3-pep517.sh:17

```

**Comment by Emru1** [comment]
thanks @lissine0 I'll check this out and try to solve this

**Comment by lissine0** [comment]
Maybe this is related? https://dev.gajim.org/gajim/gajim/-/commit/da319289

**Comment by Emru1** [comment]
It should pass check now.
Please rebase to `master`

**Comment by lissine0** [comment]
Now checks are failing for glibc flavored builds

**Comment by Emru1** [comment]
that's because repository broke down, I think

File: srcpkgs/gajim/template [hunk 1]
```diff
@@ -18,14 +18,14 @@ license="GPL-3.0-only"
 homepage="https://www.gajim.org"
 changelog="https://dev.gajim.org/gajim/gajim/-/raw/master/ChangeLog"
 distfiles="https://gajim.org/downloads/${version%.*}/gajim-${version}.tar.gz"
-checksum=4f15b2503a01bec70a6b6a203eb165ccb0b6abee513323270223a941da453f83
+checksum=649cf4ac65d096097ced9d5189521ae86554ba390370a1ffe9fdd175947f956c
 
 post_build() {
-	./pep517build/build_metadata.py -o dist/metadata
+	./make.py build --dist=unix
```
Comment (AnInternetTroll) [inline srcpkgs/gajim/template:24]: Consider adding a prefix to install icons, desktop, man pages whatever else upstream may want to in the future
```suggestion
	./make.py build --dist=unix --prefix "${DESTDIR}/usr"
```

File: srcpkgs/gajim/template [hunk 2]
```diff
@@ -18,14 +18,14 @@ license="GPL-3.0-only"
...
 }
 
 post_install() {
-	./pep517build/install_metadata.py dist/metadata --prefix=$DESTDIR/usr
+	./make.py install --dist=unix
```
Comment (AnInternetTroll) [inline srcpkgs/gajim/template:28]: Consider adding a prefix to install icons, desktop, man pages whatever else upstream may want to in the future
```suggestion
	./make.py install --dist=unix --prefix "${DESTDIR}/usr"
```

**Comment by AnInternetTroll** [comment]
I kinda skipped to 1.9.5 and that works just fine for me. It might be a good idea to update this PR as well

**Comment by Emru1** [comment]
update to 1.9.5, please

File: srcpkgs/gajim/template (hunk 2 above)
Comment (lissine0) [reply srcpkgs/gajim/template]: Thanks for the hint!

**Comment by lissine0** [comment]
@classabbyamp This (relatively-old) PR is ready for merging now.
Please take a look at it when you get the chance.
I tested it on my native architecture (x86_64-glibc) and the package works as expected.

**Comment by Emru1** [comment]
works for me :)

**Pull request merged**